### DISCUSSION TITLE
Allow for the use of Pelican within Coolify (or in conjunction with)

### DISCUSSION THREAD

**Discussion by imide** [pelican-dev/panel#511] [Ideas]
I have a server with Coolify running on it. I wish to also install Pelican but cant due to Coolify. I wish to see a "Service" within coolify that can fully install and manage Pelican (with db, cache and wings, all of it). Would be very beginner friendly too.

**Comment by TheRealMangoAPI** [comment]
Any updates on this?

**Comment by Boy132** [comment]
The panel and wings have docker images and example compose files, so it should be possible now.

If you want a one-click service directly in coolify you should create a request on [their github](https://github.com/coollabsio/coolify).
